pkg/apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -10,7 +10,6 @@ const ClusterLogForwarderKind = "ClusterLogForwarder"
 // ClusterLogForwarder is the schema for the `clusterlogforwarder` API.
 //
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-// +kubebuilder:resource:scope=cluster
 // +kubebuilder:subresource:status
 type ClusterLogForwarder struct {
```

Comment:
Why is scope removed? Although it had a type and would have worked anyway, right syntax is:
```
// +kubebuilder:resource:scope=Cluster
```

Reply:
I thought that it was the reason for some problems I saw during tests - it was unable to determine the correct operatorgroup because it's not in a namespace. Going back to namespaced seems less risky if it changes the deployment - but we should probably talk about it.

Reply:
Namespaced is fine given the arguments presented by @bparees:
* CLO is deployed to a single namespace
* CLO is only watching a single namespace
* Namespace is not the global namespace